src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -52,6 +52,12 @@ that a given scan was collected with the intended coil elements selected
 | PulseSequenceDetails        | RECOMMENDED           | [string][]                             | Information beyond pulse sequence type that identifies the specific pulse sequence used (for example, "Standard Siemens Sequence distributed with the VB17 software," "Siemens WIP ### version #.##," or "Sequence written by X using a version compiled on MM/DD/YYYY"). |
 | NonlinearGradientCorrection | RECOMMENDED           | [boolean][]                            | Boolean stating if the image saved has been corrected for gradient nonlinearities by the scanner sequence.                                                                                                                                                                |
+| MTState                     | RECOMMENDED           | [boolean][]                            | Boolean stating whether the magnetization transfer pulse is applied. Corresponds to DICOM tag (0018, 9020) `Magnetization Transfer`.                                                                                                                                      |
+| MTOffsetFrequency           | RECOMMENDED           | [number][]                             | The frequency offset of the magnetization transfer pulse with respect to the central H1 Larmor frequency in Hertz (Hz).                                                                                                                                                   |
+| MTPulseBandwidth            | RECOMMENDED           | [number][]                             | The excitation bandwidth of the magnetization transfer pulse in Hertz (Hz).                                                                                                                                                                                               |
+| MTNumberOfPulses            | RECOMMENDED           | [number][]                             | The number of magnetization transfer RF pulses applied before the readout.                                                                                                                                                                                                |
+| MTPulseShape                | RECOMMENDED           | [string][]                             | Shape of the magnetization transfer RF pulse waveform. Accepted values: `HARD`, `GAUSSIAN`, `GAUSSHANN` (gaussian pulse with Hanning window), `SINC`, `SINCHANN` (sinc pulse with Hanning window), `SINCGAUSS` (sinc pulse with Gaussian window), `FERMI`.                |
+| MTPulseDuration             | RECOMMENDED           | [number][]                             | Duration of the magnetization transfer RF pulse in seconds.                                                                                                                                                                                                               |
```

Comment:
Maybe these should be ``RECOMMENDED if MTState is `True` ``?

Reply:
Great suggestion @effigies thank you! I pushed the commit and tried to keep table pipes in order.